mix.exs

```diff
@@ -133,6 +133,7 @@ defmodule Livebook.MixProject do
     version: @version,
     logo_path: "static/images/logo.png",
     url_schemes: ["livebook"],
+    additional_paths: ["/rel/vendor/erts/bin", "/rel/vendor/elixir/bin"],
```

Comment:
Should this be:

```suggestion
    additional_paths: ["/rel/vendor/bin", "/rel/vendor/elixir/bin"],
```

Reply:
Or perhaps we should nest all of Erlang under `rel/vendor/erlang`... in any case, I think the important is that you don't need to patch the Elixir executables anymore. :)